client/cli/src/config.rs

```diff
@@ -453,13 +460,24 @@ pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
 	/// Get the node key from the current object
 	///
 	/// By default this is retrieved from `NodeKeyParams` if it is available. Otherwise its
-	/// `NodeKeyConfig::default()`.
+	/// `NodeKeyConfig::default()`..
 	fn node_key(&self, net_config_dir: &PathBuf) -> Result<NodeKeyConfig> {
 		self.node_key_params()
 			.map(|x| x.node_key(net_config_dir))
 			.unwrap_or_else(|| Ok(Default::default()))
 	}
 
+	/// Get the pre shared key from the current object
+	///
+	/// By default this is retrieved from `PreSharedKeyParams` if it is available.
+	/// Otherwise its
+	/// `PreSharedKeyKeyConfig::default()`..
```

Comment:
It's not `PreSharedKeyKeyConfig::default()` otherwise, please update this comment

Reply:
ok